target/linux/kirkwood/files-6.6/arch/arm/boot/dts/marvell/kirkwood-dns320l.dts

```diff
@@ -0,0 +1,226 @@
+/*
+ * Device Tree file for DLINK DNS-320L
```

Comment:
Are you plan send dts upstream? Debian users form doozan forum will be happy.

Reply:
Yes, I do - especially as some part of the knowledge came from that forum. :)

Reply:
@CHKDSK88 I guess all concerns so far have been fixed.

Reply:
@wigyori I found one more nit.